ega-data-api-filedatabase/src/main/java/eu/elixir/ega/ebi/downloader/service/internal/LocalEGAFileServiceImpl.java

```diff
@@ -1,110 +0,0 @@
-/*
- * Copyright 2016 ELIXIR EGA
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *      http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package eu.elixir.ega.ebi.downloader.service.internal;
-
-import com.google.gson.Gson;
-import com.netflix.hystrix.contrib.javanica.annotation.HystrixCommand;
-import eu.elixir.ega.ebi.downloader.domain.entity.File;
-import eu.elixir.ega.ebi.downloader.domain.entity.FileDataset;
-import eu.elixir.ega.ebi.downloader.domain.entity.FileIndexFile;
-import eu.elixir.ega.ebi.downloader.dto.DownloaderFile;
-import eu.elixir.ega.ebi.downloader.service.FileService;
-import lombok.extern.slf4j.Slf4j;
-import org.apache.commons.io.IOUtils;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.cache.annotation.Cacheable;
-import org.springframework.context.annotation.Profile;
-import org.springframework.stereotype.Service;
-import org.springframework.web.client.RestTemplate;
-
-import javax.transaction.Transactional;
-import java.io.IOException;
-import java.net.URL;
-import java.nio.charset.Charset;
-import java.util.Collections;
-import java.util.HashMap;
-
-// Most likely we don't need any custom implementation for LocalEGA
-// Default implementation should work "as is"
-// TODO: Remove this file after successful testing
-//@Profile("LocalEGA")
-//@Service
-//@Transactional
-@Slf4j
-public class LocalEGAFileServiceImpl implements FileService {
```

Comment:
why removed this class

Reply:
I can comment on this. It was already removed in one of the previous PRs. It's just that that commit was pulled to this PR too.

Reply:
Ok, but then there will be no implementation for LocalEGA profile of FileService Interface. Is that ok?